cmd/keycloakb/keycloak_bridge.go

```diff
@@ -1170,6 +1176,9 @@ func main() {
 		managementSubroute.Path("/realms/{realm}/groups/{groupID}").Methods("DELETE").Handler(deleteGroupHandler)
 		managementSubroute.Path("/realms/{realm}/groups/{groupID}/authorizations").Methods("GET").Handler(getAuthorizationsHandler)
 		managementSubroute.Path("/realms/{realm}/groups/{groupID}/authorizations").Methods("PUT").Handler(updateAuthorizationsHandler)
+		managementSubroute.Path("/realms/{realm}/groups/{groupID}/authorization").Methods("PUT").Handler(putAuthorizationHandler)
```

Comment:
We can't have both authorizations and authorization... do we need to keep updateAuthorizationsHandler and put(/add)AuthorizationHandler? Can't we distinguish them by the HTTP verb?

Reply:
Corrected in commit e65d1abb21021efc38ddf4adf3ef93c6273bd283. Using queryParameters